server/jetstream_api.go

```diff
@@ -1845,6 +1855,13 @@ func (s *Server) jsConsumerListRequest(sub *subscription, c *client, subject, re
 	}
 
 	streamName := streamNameFromSubject(subject)
+
+	// Clustered mode will invoke a scatter and gather.
+	if s.JetStreamIsClustered() {
+		s.startGoRoutine(func() { s.jsClusteredConsumerListRequest(acc, ci, offset, streamName, subject, reply, msg) })
```

Comment:
Same here.

Reply:
Apologies, same what?

Reply:
Same than this comment I made that you missed: https://github.com/nats-io/nats-server/pull/1819#discussion_r559226595